Makefile

```diff
@@ -8,12 +8,12 @@ SHELL := bash
 DOCKER_CMD   ?= docker
 DOCKER_ARGS  ?= run --rm --user "$$(id -u)" -v "$${PWD}:/component" --workdir /component
 
-JSONNET_FILES   ?= $(shell find . -type f -name '*.*jsonnet' -or -name '*.libsonnet')
+JSONNET_FILES   ?= $(shell find component/ -type f -name '*.*jsonnet' -or -name '*.libsonnet')
 JSONNETFMT_ARGS ?= --in-place --pad-arrays
 JSONNET_IMAGE   ?= docker.io/bitnami/jsonnet:latest
 JSONNET_DOCKER  ?= $(DOCKER_CMD) $(DOCKER_ARGS) --entrypoint=jsonnetfmt $(JSONNET_IMAGE)
 
-YAML_FILES      ?= $(shell find . -type f -name '*.yaml' -or -name '*.yml')
+YAML_FILES      ?= $(shell find class/ -type f -name '*.yaml' -or -name '*.yml')
```

Comment:
This has the risk of being missed when adding new files else where. For https://github.com/appuio/component-openshift4-monitoring/pull/10 I solved this by excluding the ones I did not care.

This is the fail save approach.  Consider to do the same.